nixos/modules/services/games/minecraft-server.nix

```diff
@@ -117,9 +122,55 @@ in {
         '';
       };
 
+      ops = mkOption {
+        type = with types;
+          listOf (submodule {
+            options = {
+              name = mkOption {
+                type = str;
+                description = "Name of the OP user";
+              };
+              uuid = mkOption {
+                type = minecraftUUID;
+                description = ''
+                  UUID of the OP user. Note that the UUIDs for a user in
+                  offline mode and online mode are different.
+                  You can use <link xlink:href="https://mcuuid.net/"/> to get a
+                  Minecraft UUID for a username in online mode.
+                  Or you can use <link xlink:href="https://www.fabianwennink.nl/projects/OfflineUUID/"/>
+                  for usernames in offline mode.
+                '';
+              };
+              level = mkOption {
+                type = int;
+                description = "Level of the OP permissions of the OP user.";
+              };
+            };
+          });
```

Comment:
`ops.json` also features a `"bypassesPlayerLimit": bool` option that I suggest adding here (defaulting to `false`).

Reply:
Busying now, probably would apply the change a few days later.